src/app/pages/task-calendar/rsync/rsync-form/rsync-form.component.ts

```diff
@@ -178,9 +177,9 @@ export class RsyncFormComponent {
         type: 'checkbox',
         name: 'rsync_preserveattr',
         placeholder: T('Preserve extended attributes'),
-        tooltip: T('Both systems <b>must</b> support <a\
-                    href="https://en.wikipedia.org/wiki/Extended_file_attributes"\
-                    target="_blank">extended attributes</a>.'),
+        tooltip: T('<a\ href="https://en.wikipedia.org/wiki/Extended_file_attributes"\
+                    target="_blank">Extended attributes</a> are\
+                    preserved, but must be supported by both systems.'),
```

Comment:
This is still a little unclear.
"preserved if supported by" is a little better.